src/EnergyPlus/DataSurfaces.cc

```diff
@@ -428,85 +428,208 @@ namespace DataSurfaces {
     // the sky for radiation from the surface
     // Fraction of sky IR coming from sky itself; 1-AirSkyRadSplit comes from the atmosphere.
 
-    Array1D<Real64> WinTransSolar; // Exterior beam plus diffuse solar transmitted through window, or
-    // window plus shade/blind, into zone (W)
-    Array1D<Real64> WinBmSolar; // Exterior beam solar transmitted through window, or
-    // window plus blind, into zone (W)
```

Comment:
Wondering why you renamed arrays that were not previously part of SurfaceWindow?

Reply:
For the naming convention/prefix we discussed about. `SurfWin`, `SurfOpaq`, `Surf`, and `Zone` to help understand their scope?

Reply:
Like, anything starts with `SurfWin` is expected to be window indexed only and inside firstWindowSurf to lastWindowSurf loop.

Reply:
Yes, I remember that discussion. I just didn't understand the scope of that went beyond the original SurfaceWindow struct. Not saying it needs to be undone, just didn't expect that.

Reply:
This convention should be propagated everywhere. The current naming convention "scheme" is terrible. As someone who doesn't actually know what should be going on, let me tell you, it really makes the code difficult to read.

Reply:
> Yes, I remember that discussion. I just didn't understand the scope of that went beyond the original SurfaceWindow struct. Not saying it needs to be undone, just didn't expect that.

:sweat_smile Yes. That went beyond that. Just after converting 30 of those beyond SurfaceWindow, I realized there are another 300 of them that better get renamed and I decided to get this partial effort merged before getting too far.